build.gradle

```diff
@@ -7,22 +7,14 @@ plugins {
   id 'jacoco'
   id 'idea'
   id 'maven-publish'
-  id 'nu.studer.plugindev' version '1.0.3'
+  id 'com.gradle.plugin-publish' version '0.9.0'
```

Comment:
This updates us to use the new plugin functionality. I will need @stormbeta to reclaim the plugin on the plugin portal if he hasn't already and also add his api credentials to the .travis.yml file. I was able to get this to work on the readytalk-js plugin by setting java system-level properties (`-D` on the gradlew call).